### ISSUE TITLE
The signature of “NewRelic.xcframework” cannot be verified.

### ISSUE THREAD

**Issue by narangkishore98** [newrelic/newrelic-ios-agent-spm#39]
[NOTE]: # ( ^^ Provide a general summary of the issue in the title above. ^^ )

## Description

Xcode 15 (beta), iOS 17.0 - Build fails while building the application. 

[NOTE]: # ( Describe the problem you're encountering. )
[TIP]:  # ( Do NOT give us access or passwords to your New Relic account or API keys! )

## Steps to Reproduce

Just upgrade the XCode to 15.0 (beta) and try to run the application. The error occurs. Similar issue was also arose with firebase https://github.com/firebase/firebase-ios-sdk/issues/11401

## Expected Behavior

The build should run smoothly without causing any issues.

[NOTE]: # ( Tell us what you expected to happen. )

## Relevant Logs / Console output

```

ProcessXCFramework /Users/username/Library/Developer/Xcode/DerivedData/ProjectName-csifkgaqfwzagadacftytziyfpxq/SourcePackages/artifacts/newrelic-ios-sdk/NewRelic/NewRelic.xcframework /Users/username/Library/Developer/Xcode/DerivedData/ProjectName-csifkgaqfwzagadacftytziyfpxq/Build/Products/Staging-Debug-iphonesimulator/NewRelic.framework ios simulator
    cd /Users/username/Workspace/additional-project/new-repo
    builtin-process-xcframework --xcframework /Users/username/Library/Developer/Xcode/DerivedData/ProjectName-csifkgaqfwzagadacftytziyfpxq/SourcePackages/artifacts/newrelic-ios-sdk/NewRelic/NewRelic.xcframework --platform ios --environment simulator --target-path /Users/username/Library/Developer/Xcode/DerivedData/ProjectName-csifkgaqfwzagadacftytziyfpxq/Build/Products/Staging-Debug-iphonesimulator

/Users/username/Library/Developer/Xcode/DerivedData/ProjectName-csifkgaqfwzagadacftytziyfpxq/SourcePackages/artifacts/newrelic-ios-sdk/NewRelic/NewRelic.xcframework:1:1: error: The signature of “NewRelic.xcframework” cannot be verified.
    note: Codesign verification failed
    note: /Users/username/Library/Developer/Xcode/DerivedData/ProjectName-csifkgaqfwzagadacftytziyfpxq/SourcePackages/artifacts/newrelic-ios-sdk/NewRelic/NewRelic.xcframework: bundle format unrecognized, invalid, or unsuitable


/Users/username/Library/Developer/Xcode/DerivedData/ProjectName-csifkgaqfwzagadacftytziyfpxq/SourcePackages/artifacts/newrelic-ios-sdk/NewRelic/NewRelic.xcframework:1:1: The signature of “NewRelic.xcframework” cannot be verified.



Build failed    2023-07-25, 10:34 AM    5.3 seconds

```

## Your Environment

XCode 15
iOS 17.0

**Comment by cdillard-NewRelic** [comment]
Can you please try with 7.4.6 and let us know the result? We've released New Relic iOS agent 7.4.6 containing a fix for this issue.

**Comment by narangkishore98** [comment]
@cdillard-NewRelic Thank you for the information. This got me working.
